src/main/pages/che-7/hands-on-guidance/proc_creating-a-workspace-by-importing-source-code-of-your-project.adoc

```diff
@@ -1,27 +1,111 @@
+* You need to have created a workspace (with plugins related to your development environment).
 
 [discrete]
-== Procedure
+== Importing from the User Dashboard into an existing workspace
+
+There are at least two ways to import a project via the User Dashboard.
+
+From the User Dashboard, select Workspaces, then select your workspace by clicking on its name, or use the gear icon. The former will link you to the workspace's *Overview* tab, while the latter will link to the *Devfile* tab where you can enter your own YAML configuration.
+
+In either case, click the *Projects* tab next.
+
+// TODO https://github.com/eclipse/che/issues/13665 remove ref to blank and zip
+From the *Projects* tab of your workspace's configuration page in the User Dashboard, click the Add Project button. You can then import a sample project, create a blank project, import from a git project, or import from a zip file.
+
+image::workspaces/workspace-config-projects.png[Add projects to workspace]
+
+[NOTE]
+====
+You can add a project to a non-running workspace, but you must start the workspace to delete it.
+====
+
+// TODO https://github.com/eclipse/che/issues/13665 remove section
+=== Create a new repository
+
+To add a link:https://github.com/che-samples/blank[blank project], give it a name and a description:
+
+image::workspaces/add-blank-project.png[Add blank project]
+
+Next, open your workspace by clicking the green *Open* button.
+
+image::workspaces/open-blank-project.png[Your empty project]
+
+=== Edit an existing repository
+
+To edit an existing repository, choose the git project or zip file, and Che will load it into your workspace.
+
+image::workspaces/add-git-project.png[Add existing git project]
+
+Next, open your workspace by clicking the green *Open* button.
 
-. Start each step with an active verb.
+// TODO https://github.com/eclipse/che/issues/13665 fix screenshot
+image::workspaces/open-git-project.png[Your git project]
 
-. Include one command or action per step.
+=== Edit commands after importing a project
 
-. Use an unnumbered bullet (*) if the procedure includes only one step.
+Once you have a project, you can add commands to it. This allows you to more easily run, debug, or launch your application in a browser.
+
+You can do this via the workpace's *Devfile* tab:
+
+image::workspaces/workspace-config-yaml.png[Add commands to workspace]
+
+Then inside the workspace, use run a command from the menu *Terminal* > *Run Task...*.
+
+image::workspaces/run-command.png[Run task]
+
+You can configure the commands from the menu *Terminal* > *Configure Tasks...*.
+
+image::workspaces/configure-command.png[Configure tasks]
+
+== Importing to a running workspace with `git clone` command
+
+In addition to the approach above, where you can add projects via your workspace's *Projects* or *Devfile* tab, you can simply start a workspace, and use `git clone` from the command palette.
```

Comment:
If you remove the first half of the sentence, then maybe we should start with "Start a workspace, then use the `git clone` command..." rather than "You can..."

Reply:
+1 Don't use "you can".

Reply:
The phrase "You can" appears 400 times in 122 of the files in the Che docs, but sure. I can make at least the NEW docs more compliant with the rules. :)